test/v2/controllers/CreateAmendForeignPropertyAnnualSubmissionControllerSpec.scala

```diff
@@ -125,108 +66,90 @@ class CreateAmendForeignPropertyAnnualSubmissionControllerSpec
           .wrap((), CreateAmendForeignPropertyAnnualSubmissionHateoasData(nino, businessId, taxYear))
           .returns(HateoasWrapper((), Seq(testHateoasLink)))
```

Comment:
In the ControllerBaseSpec we have testHateoasLinks and testHateoasLinksJson. So, we can remove the testHateoasLink created in this class and instead use those. For this test that would mean we update line 67-69 to be 

`
          .returns(HateoasWrapper((), testHateoasLinks))

        runOkTest(expectedStatus = OK, maybeExpectedResponseBody = Some(testHateoasLinksJson))
`